r11701/default_common_inlists/binary/src/run_star_extras.f

```diff
@@ -37,7 +37,7 @@ module run_star_extras
   logical :: late_AGB_check = .false.
   logical :: post_AGB_check = .false.
   logical :: pre_WD_check = .false.
-  real(dp) :: current_wind_prscr = -1d0
+  real(dp) :: current_wind_prscr(2)
```

Comment:
Please think about an alternative initialization with the values being -1, that we have a defined value in case it won't be changed.

Reply:
Thanks, updated to initialize with values of -1